string command
float32[] values
